weave/wandb_interface/wandb_lite_run.py

```diff
@@ -51,6 +51,8 @@ def __init__(
         group: typing.Optional[str] = None,
         _hide_in_wb: bool = False,
         _use_async_file_stream: bool = False,
+        _always_create_project: bool = True,
```

Comment:
the term `always` is not clear to me - what does it mean to set this to false?

Reply:
fair -- I can remove `always`.